Updating description fields (#6852)
fishtown-analytics/docs.getdbt.com: website/docs/docs/build/sql-models.md

---
title: "SQL models"

> description: "Learn how to use SQL models when building in dbt."

Author: ```suggestion
description: "SQL models are the building blocks of your dbt project."
```
